project-base/src/Controller/Front/CartController.php

```diff
@@ -332,4 +339,34 @@ public function deleteAction(Request $request, $cartItemId)
 
         return $this->redirectToRoute('front_cart');
     }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @param int $cartItemId
+     */
+    public function deleteAjaxAction(Request $request, $cartItemId)
+    {
+        $cartItemId = (int)$cartItemId;
+        $token = new CsrfToken('front_cart_delete_' . $cartItemId, $request->query->get('_token'));
+
+        if ($this->get('security.csrf.token_manager')->isTokenValid($token)) {
```

Comment:
I suggest simplifying this code according to the [Symfony documentation](https://symfony.com/doc/current/security/csrf.html#generating-and-checking-csrf-tokens-manually) to

```php
$token = $request->query->get('_token');

if ($this->isCsrfTokenValid('front_cart_delete_' . $cartItemId, $token)) {
```

it's easier to read and we don't use service locator (get service from the container by id), what do you think?